pkg/querier/codec/protobuf_codec_test.go

```diff
@@ -147,6 +149,7 @@ func TestProtobufCodec_Encode(t *testing.T) {
 			},
 		},
 		{
+			name: "matrix",
```

Comment:
nit: missing name for this matrix test https://github.com/cortexproject/cortex/pull/6345/files#diff-9c2062cda65af13cf43cc9371cae0424744499ad5ca83597ea2bf4df2437f4d8R106

seems like it is the same as https://github.com/cortexproject/cortex/pull/6345/files#diff-9c2062cda65af13cf43cc9371cae0424744499ad5ca83597ea2bf4df2437f4d8R186?

Reply:
Yeah.. let's add a name and delete the duplicate one. Thanks!